packages/shell-api/src/shell-instance-state.ts

```diff
@@ -481,6 +487,10 @@ export default class ShellInstanceState {
 
   // eslint-disable-next-line @typescript-eslint/require-await
   async getDefaultPrompt(): Promise<string> {
+    if (this.connectionInfo?.extraInfo?.is_stream) {
```

Comment:
Aren't the other parts of the prompt relevant here? At least the dbName .. or in streams `use somedb` should just be disabled?

Reply:
Database isn't really relevant for streams instance. There isn't any database. We removed the database name from the connection URL for the same reason. Good question about `use samedb`, I am not actually sure. Will need to take it up with the PMs. Maybe the global `db` should not be added even.

Reply:
yeah good point about `db`, please loop in also the PMs of the shell